content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -30,34 +30,62 @@ Using the latest version of kubectl helps avoid unforeseen issues.
 
 ### Install kubectl binary with curl on Linux
 
-1. Download the latest release with the command:
+1. Download the latest stable release with the command:
 
+    ```bash
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
     ```
-    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
-    ```
 
-    To download a specific version, replace the `$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)` portion of the command with the specific version.
+    {{< note >}}
+To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
+
+For example, to download version {{< param "fullversion" >}} on Linux, type:
+
+```bash
+curl -LO https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/linux/amd64/kubectl
+```
+    {{< /note >}}
+
+    **Validate the binary (optional)**
```

Comment:
I'd make this a heading, with the “optional” omitted from the fragment identifier. Did you consider that and decide against it @nate-double-u? (if so, what helped you decide?)

Reply:
I had considered that, but because its mid-list I thought it would be better not to break up the list with a header. I figured any header is higher on the hierarchy than any list item, so I thought it might break the flow.

Reply:
Hi @nate-double-u . Seems like the checksum validation belongs with the "get kubectl" instructions. You want
to validate the binary before you proceed to make the binary executable and move kubectl to a different location
on your file system.
I think you could list the "checksum" instructions as a new list item. Something like:

```
1. Validate the kubectl binary (optional)

    * Download the kubectl checksum file:
    ... do something
    * Validate the kubectl binary against the checksum file:
    ... do something
       The output is similar to:
       "kubectl OK!

1. Make the kubectl binary executable.
```

nit: Check that you have the code block lined up in the list item.

Reply:
I like this option @kbhawkey, I'll try it this way and see how it works.

Reply:
https://deploy-preview-25259--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/#install-kubectl-binary-with-curl-on-linux

and:
* https://deploy-preview-25259--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/#install-kubectl-binary-with-curl-on-macos
* https://deploy-preview-25259--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/#install-kubectl-binary-with-curl-on-windows